src/components/alert/alert.e2e.ts

```diff
@@ -192,6 +192,19 @@ describe("calcite-alert", () => {
     expect(container).toHaveClass("top-end");
   });
 
+  it("has aria-live attribute set to assertive on autoDimiss", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+    <calcite-alert placement="top-end" auto-dismiss>
+    ${alertContent}
+    </calcite-alert>`);
+
+    const alert = await page.find("calcite-alert");
+    const priority = alert.getAttribute("aria-live");
+
+    expect(priority).toBe("assertive");
+  });
+
```

Comment:
@anveshmekala can you add another test using the `accessible` test in `commonTests`? 

This will run the HTML in aXe.

Reply:
Currently we have a [test](https://github.com/Esri/calcite-components/pull/4013/files#diff-c4f0b7905696c89b9d919db65d83f2fba7f2a25c40b8682fb79ea29fbd19a4a1L14) to check accessibility . Do we need to add another?

Reply:
Yeah, that doesn't run the HTML through aXe. I think we need to do that to verify that it is accessible.